docs/resources/techresources/techstack.md

```diff
@@ -19,13 +19,13 @@ products Web Developers may choose from in the JavaScript ecosystem.
 
 | Technology              | Frontend | Backend | Database |
 | ----------------------- | -------- | ------- | -------- |
-| Basic                   |          |         |          |
+| **__BASIC TECH__**      |          |         |          |
 | HTML                    | ✓        |         |          |
 | CSS                     | ✓        |         |          |
 | JavaScript              | ✓        |         |          |
 | Express                 |          | ✓       |          |
 | GitHub Pages            | ✓        |         |          |
-| Intermediate            |          |         |          |
+| **__INTERMEDIATE TECH__** |          |         |          |
 | Sass                    | ✓        |         |          |
 | jQuery                  | ✓        |         |          |
```

Comment:
maybe add React and Nextjs since they are used a lot in the solo projects and voyage teams
not sure about jquery, feels outdated
Firebase and appwrite is probably not advanced